extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.Icons-vector.scss

```diff
@@ -391,6 +391,12 @@
 .oo-ui-icon-media {
 	@include ve-icon(media, wikia);
 }
+.oo-ui-icon-image {
+	@include ve-icon(image, wikia);
+}
+.oo-ui-icon-video {
```

Comment:
I've searched the code, and I am pretty sure it is not used. I suppose there is a chance it is used with a jquery.css call somewhere outside scss files, but I think it is unlikely.

I'll remove it.